stdlib/list.gr

```diff
@@ -262,42 +274,39 @@ let rec take = (n, list) => {
     match ((n, list)) {
       (_, []) => list,
       (n, _) when n == 0 => [],
-      (n, [first, ...rest]) => [first, ...take(n-1, rest)]
+      (n, [first, ...rest]) => [first, ...take(n - 1, rest)],
     }
   }
 }
 
 let rec takeWhile = (p, list) => {
   match (list) {
     [] => [],
-    [first, ...rest] => if (p(first)) [first, ...takeWhile(p, rest)] else []
+    [first, ...rest] => if (p(first)) [first, ...takeWhile(p, rest)] else [],
   }
 }
 
 let rec find = (fn, list) => {
   match (list) {
     [] => None,
-    [first, ...rest] => if (fn(first)) Some(first) else find(fn, rest)
+    [first, ...rest] => if (fn(first)) Some(first) else find(fn, rest),
   }
 }
 
 let findIndex = (fn, list) => {
   let rec findItemIndex = (l, index) => {
     match (l) {
       [] => None,
-      [first, ...rest] => if (fn(first)) Some(index) else findItemIndex(rest, index + 1)
+      [first, ...rest] =>
+        if (fn(first)) Some(index) else findItemIndex(rest, index + 1),
     }
   }
   findItemIndex(list, 0)
 }
 
 let product = (a, b) => {
   let mut list = []
-  forEach((aItem) => {
-    forEach((bItem) => {
-     list = [(aItem, bItem), ...list]
-    },b)
-  }, a)
+  forEach(aItem => {forEach(bItem => {list = [(aItem, bItem), ...list]}, b)}, a)
```

Comment:
I either want the brackets removed or to always split as a block. Looks like it happens twice in here 😛